docs/capabilities/account.md

```diff
@@ -1,10 +1,31 @@
 # Account management
 
-Account management is one of the core capabilities provided by AlgoKit Utils. It allows you to create mnemonic, rekeyed, multisig, transaction signer, idempotent KMD and environment variable injected accounts that can be used to sign transactions as well as representing a sender address at the same time. This significantly simplifies passing around sender/signer within and into AlgoKit Utils.
+Account management is one of the core capabilities provided by AlgoKit Utils. It allows you to create mnemonic, rekeyed, multisig, transaction signer, idempotent KMD and environment variable injected accounts that can be used to sign transactions as well as representing a sender address at the same time. This significantly simplifies management of transaction signing.
 
-## `SendTransactionFrom`
+## `AccountManager`
 
-Any AlgoKit Utils function that needs to sign/send a transaction will take a [`SendTransactionFrom`](../code/modules/types_transaction.md#sendtransactionfrom) object, which represents an account that combined a sender and signer and is a type union between the following types:
+The [`AccountManager`](../code/classes/types_account_manager.AccountManager.md) is a class that is used to get, create, and fund accounts and perform account-related actions such as funding. The `AccountManager` also keeps track of signers for each address so when using the [`AlgokitComposer`](./algokit-composer.md) to send transactions, a signer function does not need to manually be specified for each transaction - instead it can be inferred from the sender address automatically!
+
+To get an instance of `AccountManager` you can get it from either [`AlgorandClient`](./algorand-client.md) via `algorand.account` or instantiate it directly (passing in a [`ClientManager`](./client.md)):
+
+```typescript
+import { AccountManager } from '@algorandfoundation/algokit-utils/types/account-manager'
+
+const accountManager = new AccountManager(clientManager)
+```
+
+## `TransactionSignerAccount`
+
+The core internal type that holds information about a signer/sender pair for a transaction is [`TransactionSignerAccount`](../code/interfaces/types_account.TransactionSignerAccount.md), which represents an `algosdk.TransactionSigner` (`signer`) along with a sender address (`addr`) as the encoded string address.
+
+Many methods in `AccountManager` expose a `TransactionSignerAccount`.
+
+### `SendTransactionFrom`
```

Comment:
A lot of the existing docs link to this jump link so important to keep it for now, added the legacy note.